RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -79,8 +79,8 @@
 
     # EM Scale calibrations
     layerWeights = dEdX_weights,
-    thicknessCorrection = cms.vdouble(0.964,0.920,0.909), # 100, 200, 300 um
-    
+    thicknessCorrection = cms.vdouble(1.132,1.092,1.084), # 100, 200, 300 um
```

Comment:
@clelange @felicepantaleo : any documentation of the modifications originated by this pull request, together with an explanation of the same which attest they are welcome, is welcome.

If you feel more at ease, you could even split this PR in two, taking aside the part which affects the other parts of CMSSW (and it is therefore reflected in the standard wfs outputs). But for what I am concerned, provided that the modifications are documented and deemed acceptable I can sign this PR for reco.